moped-editor/src/views/projects/projectsListView/ProjectsListViewQueryConf.js

```diff
@@ -101,26 +101,26 @@ export const ProjectsListViewQueryConf = {
       },
       type: "string",
     },
-    "moped_proj_personnel (where: {status_id: { _eq:1 }}) { moped_user { first_name last_name } moped_project_role { project_role_name }}": {
+    project_team_members: {
       searchable: false,
       sortable: false,
-      stringify: true,
       label: "Team Members",
       width: "20%",
       filter: value => {
-        const parsedJson = JSON.parse(value)
+        if (value === " :") {
+          return ""
+        }
+        let namesArray = value.split(',')
         let uniqueNames = {}
         let personnel = []
-
-        parsedJson.forEach(person => {
-          let fullName = person.moped_user.first_name + " " + person.moped_user.last_name
+        namesArray.forEach(person => {
+          let [fullName, projectRole] = person.split(":")
           if (uniqueNames[fullName]) {
-            uniqueNames[fullName] = uniqueNames[fullName] + `, ${person.moped_project_role.project_role_name}`
+            uniqueNames[fullName] = uniqueNames[fullName] + `, ${projectRole}`
           } else {
-            uniqueNames[fullName] = person.moped_project_role.project_role_name
+            uniqueNames[fullName] = projectRole
           }
         })
-
         for (const [key, value] of Object.entries(uniqueNames)) {
           personnel.push(`${key} - ${value}`)
         }
```

Comment:
This is very clever, I like how the value turns into a string and then the string gets broken down and parsed. There are however certain things that need change, but not deal-breakers:

1. The comma notation, that's important for me because it helps me know whenever an expression ends.
2. A few variables (if not all) can be made `const`, they are not modified and do not need to be `let` variables.
3. Great use of `namesArray.forEach` it looks great, but the `Object.entries` loop can be improved.